tools/ci/src/main.rs

```diff
@@ -16,8 +16,9 @@ bitflags! {
     }
 }
 
-const CLIPPY_FLAGS: [&str; 8] = [
+const CLIPPY_FLAGS: &[&str] = &[
     "-Aclippy::type_complexity",
+    "-Aclippy::module_inception",
```

Comment:
Is this supposed t obe changed? Not sure why this is in this PR.

Reply:
This is to solve problem below.
```
mod schedule {
    mod schedule {
    }
}
```
The solution is from https://github.com/alice-i-cecile/bevy/pull/98 .

Reply:
I think we should just do it on the one line where it's needed.

Reply:
Agreed, this is a global setting and changing our CI like this would be a larger discussion. Perhaps just add a `#[allow(clippy::module_inception)]` where we need it, as @hymm suggested.

Reply:
Done